packages/steveo/package.json

```diff
@@ -9,7 +9,7 @@
   "repository": "git@github.com:ordermentum/steveo.git",
   "scripts": {
     "test": "NODE_ENV=test nyc yarn spec",
-    "spec": "tsc && yarn mocha",
+    "spec": "tsc && yarn mocha --require ts-node/register --bail=false 'test/**/*_test.ts'",
```

Comment:
can these options be put in the mocha opts file

Reply:
doh 🤦 , they largely are.... good call